src/DataProccesing.java

```diff
@@ -0,0 +1,95 @@
+import java.io.*;
+
+public class DataProccesing {
+
+    /**
+     * @param login    - written by user;
+     * @param password - written by user;
+     */
+    public static void dataSaving(String login, String password) {
```

Comment:
**All** method should be non-static. Instead create 2 objects in Main before implementation and use objects methods

Also - since 2 objects will be non-static you don't need to open and close file each time. You can just open if during object creation. Don't forget to close resources in Main at the end. For example you can use Autocloseable interface.